var/spack/repos/builtin/packages/openimagedenoise/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Openimagedenoise(Package):
+    """Intel Open Image Denoise is an open source library of high-performance,
+    high-quality denoising filters for images rendered with ray tracing.
+    Open Image Denoise is part of the Intel® oneAPI Rendering Toolkit and is released
+    under the permissive Apache 2.0 license."""
+
+    homepage = "https://www.openimagedenoise.org/"
+    url = "https://github.com/OpenImageDenoise/oidn/archive/v1.2.3.tar.gz"
+
+    # maintainers("github_user1", "github_user2")
+
+    version("1.4.3", sha256="37941f260af212183579efb6b1e07e4075e3ab9cbd63120afd007e62367efef6")
+    version("1.4.2", sha256="742de28e2c018dc47819aaeac039e504df1ce660ed72aa0b49009343c0a9aae1")
+    version("1.4.1", sha256="db022565ffe46266188992e1db68d11c94f799c37e25996f5aee44bebffcf079")
+    version("1.4.0", sha256="4f285b0f58d3aa4419afd1bf57784416087119dc620dcf1a6b9fa19a27b32c5b")
+    version("1.3.0", sha256="a6b30d7aa3ea81490431dba823dab475daad73a7124940560624742126db7274")
+    version("1.2.4", sha256="f847c244417e68ba46149663055f163188cc2ec62308fe040ec943df184a2c40")
+    version("1.2.3", sha256="9ff7f7b5c442bd56be1ae89f8c45ae1972498b854f7175705ff520bb2c5cb25b")
+
+    depends_on("ispc")
+    depends_on("python@3:")
```

Comment:
Build and link or just build?

Reply:
good catch, it is build and test only. ispc is also only a build dep fixed that too.